docs/components/ColorTile.js

```diff
@@ -1,19 +1,23 @@
 // @flow strict
 import { type Node } from 'react';
-import { Box, Text } from 'gestalt';
+import { Box, Text, useColorScheme } from 'gestalt';
 // $FlowExpectedError[untyped-import]
 import tokens from 'gestalt-design-tokens/dist/json/variables.json';
+// $FlowExpectedError[untyped-import]
+import darkModeTokens from 'gestalt-design-tokens/dist/json/variables-dark.json';
 
 type Props = {|
   fullTokenName: string,
   description: string,
   number?: number,
-  textColor?: 'dark' | 'light',
+  textColor?: 'dark' | 'light' | 'default' | 'inverse',
```

Comment:
nit: unless restricting the options is desired, you can also just type this as
```js
$ElementType<React$ElementConfig<typeof Text>, 'color'>
```
to future-proof it

Reply:
ahh ok, it might be worth restricting it for now, since only those options should be used on top of colored backgrounds, but I'll keep it in mind